Kajabi/sage-lib
# Dropdown Custom Content

## Description

- Adds the ability to pass custom content to a dropdown and relies less on items to have to be links or buttons.
- Adds the ability to close the custom dropdown panel by applying a `data-sage-dropdown-exit` data attribute.
- Adjusts the z-index of dropdowns in order to fix the overlapping issues with other elements. (eg. tables, panel controls, etc...)

## Test notes

- Visit http://localhost:4000/pages/breakout/object/dropdown
- Verify the dropdown in the "Dropdown w/ Custom Content" section, displays the custom content.
- Click the button in the custom dropdown to verify the panel closes (the button has data attribute applied.)
--
- Visit http://localhost:4000/pages/breakout/object/panel_controls#
- Verify dropdowns no longer overlap or appear behind other elements.
--
- Setup bridge with kajabi-products.
- Under People ensure that all dropdowns at the top for segments and filters work as expected, and that known z-index/layering bugs are resolved.

### Estimated impact
- [x] (LOW) Updates dropdown to allow custom content to be used within a dropdown and to use a data-attribute to close the custom panel.
- [x] (MED) Introduces a z-index fix for dropdowns to fix overlapping issues Under People ensure that all dropdowns at the top for segments and filters work as expected and that known z-index/layering bugs are resolved.
